core/types/outputpayload/transfercrosschainfee_test.go

```diff
@@ -0,0 +1,65 @@
+package outputpayload
+
+import (
+	"bytes"
+	"github.com/elastos/Elastos.ELA/common"
```

Comment:
Attention to the import format, it's hidden in the goland IDE.

Reply:
Already changed as required